### DISCUSSION TITLE
Welcome to hfeasy Discussions!

### DISCUSSION THREAD

**Discussion by ljalves** [ljalves/hfeasy#28] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by darrellm77** [comment]
I have a Ankuoo NEO light switch.
It has unconfigured itself, the Ankuoo NEO APP no longer works , thus I cannot use any APP provided FW update process.
Where are tutorials to serially flash this thing to HFEASY ??

**Comment by mercs759** [comment]
Hello,
I have a Lidl / Silvercrest SWS-A1 (PA-GE1-01SW), it had original silvercrest fw 1.0.04b, 
I did the first wifi connection of SWS-A1 to my router using icomen 1.20 apk, (Smart Link tecnique)
Then i gave a static IP to my plug.

successfully flashed HFeasy v1.1 via web (Chrome v112.0.5615.138 + windows10)
everything seemed ok. 
Plug worked in hfeasy default wifi mode (AP+STA)

I set device to "SWS-a1", did a few on and off via http interface.

Then I used HFeasy web interface (black and white one) to set Wifi to "station only" , 
AP went away: ok. Station worked,

But then I hit save and restart.
And now plug doesn't connect to my router anymore. 

I rebooted with hardare button, and HFeasy AP came on , but STA didn't connect

both HFeasy webinterface (black and white) and the original (blue) interface report that wifi mode is still on "ap+sta" (it's not been modified by my last command (sta only), it seems.
My routers SSID and ecrypt & password are still displayed in both web interfaces.
But plug doesnt connect as client.

So i tried again to put the ACplug in "STA only" mode, this time using the classic blue interface. 
Webconfig said to restart. I restarted. 
Plug is bricked. No way to bring the AP on.
